site/pages.json

```diff
@@ -546,15 +556,15 @@
           "description": "Laufen",
           "children": [
             {
-            "name": "Laufen",
-            "destination": "trainingsplan",
-            "file": "triathlon/laufen/trainingsplan/index.html",
-            "title": "Laufen",
-            "photoUrl": "https://svm-dh.herokuapp.com/assets/images/running1.jpg",
-            "description": "Laufen",
-            "children": [
-            ]
-          }
+              "name": "Laufen",
+              "destination": "trainingsplan",
+              "file": "triathlon/laufen/trainingsplan/index.html",
+              "title": "Laufen",
+              "photoUrl": "https://svm-dh.herokuapp.com/assets/images/running1.jpg",
+              "description": "Laufen",
+              "children": [
+              ]
```

Comment:
no need for that - all trainingsplan pages can be removed - in the branch improvements/triathlon_startpage I did remove those already